src/platforms/javascript/common/performance/index.mdx

```diff
@@ -1,218 +1,188 @@
 
 ```html
 <html>
-<head>
-  <meta name="sentry-trace" content="{{ span.toTraceparent() }}">
-  <!-- ... -->
+  <head>
+    <meta name="sentry-trace" content="{{ span.toTraceparent() }}" />
+    <!-- ... -->
+  </head>
+</html>
 ```
 
-The `name` attribute must be the string `"sentry-trace"` and the `content` attribute must be generated by your backend's Sentry SDK using `span.toTraceparent()` (or equivalent, depending on the backend platform).
+The `name` attribute must be the string `"sentry-trace"` and the `content` attribute must be generated by your backend's Sentry SDK using `span.toTraceparent()` (or equivalent, depending on the backend platform). This guarantees that a new and unique value will be generated for each request.
 
-The `span` reference is either the transaction that rendered the page or any of its child spans. It defines the parent of the `pageload` transaction.
+The `span` reference is either the transaction that serves the HTML, or any of its child spans. It defines the parent of the `pageload` transaction.
 
-For the correct association of transactions in a trace, make sure your backend will render a new and unique value for the `content` attribute for every request. Reusing the same value will cause unrelated traces to incorrectly share the same `trace_id`.
+Once the data is included in the `<meta>` tag, our `BrowserTracing` integration will pick it up automatically and link it to the transaction generated on pageload. (Note that it will not get linked to automatically-generated `navigation` transactions, that is, those which don't require a full page reload. Each of those will be the result of a different request transaction on the backend, and therefore should have a unique `trace_id`.)
 
-<Alert level="info" title="Note"><markdown>
+### Navigation and Other XHR Requests
 
-JavaScript code running on a browser cannot read the response headers of the current page. Therefore, the `<meta>` tag is used to propagate tracing metadata from the backend instead of the `sentry-trace` HTTP header described earlier.
+Once a page is loaded, any requests it makes and any responses your backend generates as a result are linked through a request/response header.
 
-</markdown></Alert>
+As is the case with the `<meta>` tag discussed above, the header's name is `sentry-trace` and its value is obtained by calling `span.toTraceparent()` (or the equivalent), where `span` is either the relevant transaction or any of its children.
 
-### beforeNavigate
+All of Sentry's tracing-related integrations (`BrowserTracing`, `Http`, and `Express`) either generate or pick up and propagate this header automatically as appropriate, for all transactions and spans which they generate. You can also attach and read the header yourself, in any case in which you've manually created either a transaction or a span and it makes sense to do so.
```

Comment:
This is currently a lie, in that it's a bug that this is not true, and is on our backlog to fix. Should I take it out or leave it and just fix the bugs (hopefully) soon?

Reply:
Perhaps comment it out for now?